src/reducers/index.js

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { combineReducers } from 'redux';
+import books from './books';
+
```

Comment:
[Optional] Quick one:
By the time you have more than one reducers to combine, this style might look unprofessional
```
const rootReducer = combineReducers({
  books,
});
```